tools/proofers/toolbox.js

```diff
@@ -1,4 +1,4 @@
-/*global */
+/* eslint-disable no-unused-vars */
```

Comment:
hmm, I wonder if more exported are needed? Since we switched to modules though, we should rely on exports or window property assignments to resolve this?